run--01a--prep-geo-data.Rmd

```diff
@@ -533,17 +664,101 @@ if (exists("my_aux_geo")) {
              County = case_when(max(in_aux_flag) == 0 ~ County,
                                 in_aux_flag      == 0 ~ paste(County, glue("(Non-{my_aux_overall_name})")),
                                 in_aux_flag      == 1 ~ paste(County, glue("({my_aux_overall_name})"))),
-             pct_area = ifelse(is.na(pct_area), 1, pct_area))
+             pct_area = ifelse(is.na(pct_area), 1, pct_area)) %>% 
+      mutate(County = ifelse(County %like% "DuPage", "DuPage", County))
     
     if (FALSE) {
       cbind(
         geo_crosswalk     %>% filter(chicago_flag == 1) %>% select(GEOID, pct_area) %>% arrange(GEOID), 
         geo_crosswalk_aux %>%                               select(GEOID, pct_area) %>% arrange(GEOID))
     }
   }
+} else {
+  geo_crosswalk <- geo_crosswalk %>% 
+    mutate(pct_area = 1)
 }
 ```
 
+```{r add Birth to Five Council for Illinois}
```

Comment:
Not a fan of this being in `main`. Remind me why this can't be constructed separately and submitted as an `aux` geography?